src/main/java/adris/altoclef/tasks/resources/CollectSticksTask.java

```diff
@@ -0,0 +1,128 @@
+package adris.altoclef.tasks.resources;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.Debug;
+import adris.altoclef.TaskCatalogue;
+import adris.altoclef.tasks.CraftInInventoryTask;
+import adris.altoclef.tasks.ResourceTask;
+import adris.altoclef.tasks.construction.DestroyBlockTask;
+import adris.altoclef.tasks.movement.PickupDroppedItemTask;
+import adris.altoclef.tasks.movement.TimeoutWanderTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.CraftingRecipe;
+import adris.altoclef.util.ItemTarget;
+import adris.altoclef.util.RecipeTarget;
+import adris.altoclef.util.helpers.ItemHelper;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class CollectSticksTask extends ResourceTask {
+
+    private final int _targetCount;
+
+    private static final Item[] _itemsToPickup;
+    static {
+        List<Item> itemsToPickup = new ArrayList<>(Arrays.asList(ItemHelper.LOG));
```

Comment:
You shouldn't need to have a list of itemsToPickup. If you think about it, we only need to check if dead bush is an option. Otherwise, just do what we would have done in Task Catalogue.

`        return new CraftInInventoryTask(new RecipeTarget(Items.STICK,_targetCount, CraftingRecipe.newShapedRecipe("sticks", new ItemTarget[]{t("planks"), t(null), t("planks"), t(null)},4)));
`